Policies/Kubernetes/block-host-namespace/azurepolicy.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "displayName": "Do not allow sharing of the Host Namespace in a Kubernetes Cluster",
```

Comment:
Host namespaces here https://kubernetes.io/docs/concepts/policy/pod-security-policy/#host-namespaces network and ports check as well. As this policy is only checking hostPID and hostIPC, can you update the displayName to something like:
Do not allow sharing of host process ID and IPC namespaces.